frontend/src/libs/components/meditation-player/components/audio-player/controls/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+.controls {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 40px;
+}
+
+button:nth-child(1) {
```

Comment:
According to rule B10 we can't use tag names in styles. Because of we can't directly send to props className it was asked to wrap elements with div or kind of it. Also I think it will be better approach to wrap all buttons in div with gap 40px and wrap in it center 3 buttons with gap 21px